docker-compose.yml

```diff
@@ -8,12 +8,12 @@
 # and additionally use the Django development server which is
 # unsuitable for production.
 
-version: "3.6"
+version: "3.8"
```

Comment:
For future reference, there isn't really a point in changing this as its deprecated

https://github.com/compose-spec/compose-spec/blob/master/spec.md#version-top-level-element

> A Compose implementation SHOULD NOT use this version to select an exact schema to validate the Compose file, but prefer the most recent schema at the time it has been designed.

Maybe there are still some tools that are not respecting this advice?

Reply:
My compose does use the behavior you described above, so I'd be happy to remove it. The reason I bumped it is because our current compose is using features that weren't even introduced in 3.6, and I wanted to fix the linting issues.

Reply:
Does `docker-compose` still rely on it (notice the dash)? Maybe wait until that's completely phased out? I don't really know. It doesn't hurt to leave it for now I guess.